sequencer/src/lib.rs

```diff
@@ -596,12 +596,8 @@ pub mod testing {
     const STAKE_TABLE_CAPACITY_FOR_TEST: u64 = 10;
 
     pub async fn run_test_builder<const NUM_NODES: usize>(
-        mut url: Url,
-        port: Option<u16>,
+        url: Url,
```

Comment:
I think I meant the opposite suggestion. We should pick an unused port (or have one passed in) and the url we use for the builder is just `localhost:{that port}`. Now it looks like we are using the same URL for the builder and L1 in most tests which isn't going to work